src/ui.cpp

```diff
@@ -696,8 +696,21 @@ void UpdateDialog::OnRunInstaller(wxCommandEvent&)
     }
 }
 
+static inline bool ends_with(std::wstring const& value, std::wstring const& ending)
+{
+    if (ending.size() > value.size()) return false;
+    return std::equal(ending.rbegin(), ending.rend(), value.rbegin());
+}
+
```

Comment:
You already have `wxString::EndsWith()` at your disposal.

Reply:
Roger.